tests/tests.js

```diff
@@ -1,45 +1,127 @@
 const { expect } = require('chai')
+const chai = require('chai')
 const { describe, it } = require('mocha')
 const employee = require('./employee')
 const products = require('./products')
 const pricing = require('../pricing')
+const sinon = require('sinon')
+const sinonChai = require('sinon-chai')
 
-describe('calculateProductPrice', () => {
-  it('returns the price for a voluntary life product for a single employee', () => {
-    const selectedOptions = {
-      familyMembersToCover: ['ee'],
-      coverageLevel: [{ role: 'ee', coverage: 125000 }],
-    }
-    const price = pricing.calculateProductPrice(products.voluntaryLife, employee, selectedOptions)
 
-    expect(price).to.equal(39.37)
+chai.use(sinonChai)
+
+describe('pricing', () => {
+  describe('formatPrice', () => {
+    it('returns the price given, truncated to two decimal places', () => {
+      const formattedPrice = pricing.formatPrice(15.335)
+      expect(formattedPrice).to.equal(15.33)
+    })
+    it('returns the price with two decimal places when a whole number is provided', () => {
+      const formattedPrice = pricing.formatPrice(15)
+      expect(formattedPrice).to.equal(15.00)
+    })
+  })
+
+  describe('getEmployerContribution', () => {
+    it('returns the contribution amount from the product when the contribution type is dollars', () => {
+      const contribution = pricing.getEmployerContribution(products.longTermDisability.employerContribution, 15.33)
+
+      expect(contribution).to.equal(10)
+    })
+    it('returns the calculated contribution (percent of price) when the contribution type is a percentage', () => {
+      const contribution = pricing.getEmployerContribution(products.voluntaryLife.employerContribution, 100)
+
+      expect(contribution).to.equal(10)
```

Comment:
This is a good test, but consider using a different number for your price. Since you did 100 and got 10 you can't be absolutely certain that it did percent and not dollars.